userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -273,21 +273,15 @@ libsinsp::events::set<ppm_sc_code> expected_sinsp_state_ppm_sc_set = {
 #endif
 };
 
-/* This test asserts that `enforce_sinsp_state_ppm_sc` correctly retrieves
- * the `libsinsp` state ppm_sc set.
- */
-TEST(interesting_syscalls, enforce_sinsp_state_basic)
+TEST(interesting_syscalls, sinsp_state_sc_set)
 {
 	auto state_ppm_sc_set = libsinsp::events::sinsp_state_sc_set();
-
-	ASSERT_EQ(expected_sinsp_state_ppm_sc_set.size(), state_ppm_sc_set.size());
+	ASSERT_PPM_SC_CODES_EQ(expected_sinsp_state_ppm_sc_set, state_ppm_sc_set);
 	ASSERT_TRUE(expected_sinsp_state_ppm_sc_set.equals(state_ppm_sc_set));
+	ASSERT_EQ(expected_sinsp_state_ppm_sc_set.size(), state_ppm_sc_set.size());
```

Comment:
Here we are asserting almost the same thing, not sure if we want to preserve all three assertions, but if we want probably we can move the last 2 assertions in the macro `ASSERT_PPM_SC_CODES_EQ` in this way we can write just one line. Same for all other usages in the PR :)

Reply:
Great idea, incorporated :)